command/crypto/winpe/winpe.go

```diff
@@ -0,0 +1,191 @@
+package winpe
+
+import (
+	"debug/pe"
+	"encoding/asn1"
+	"encoding/binary"
+	"encoding/pem"
+	"github.com/pkg/errors"
+	"github.com/smallstep/cli/errs"
+	"github.com/urfave/cli"
+	"go.mozilla.org/pkcs7"
+	"io"
+	"os"
+	"path"
+)
+
+// Command returns the winpe subcommand.
+func Command() cli.Command {
+	return cli.Command{
+		Name:        "winpe",
+		Usage:       "Extract certificates and verify Windows Portable Executable files",
+		UsageText:   "step crypto winpe <subcommand> [arguments] [global-flags] [subcommand-flags]",
+		Description: `**step crypto winpe** command group provides facilities to extract certificates and verify Windows Portable Executable files.`,
+		Subcommands: cli.Commands{
+			extractCommand(),
+		},
+	}
+}
+
+var ErrCodeSignCertsNoCertificateTableFound = errors.New("No Certificate Table found")
+
+func extractCommand() cli.Command {
+	return cli.Command{
+		Name:      "extract",
+		Action:    cli.ActionFunc(extractPEAction),
+		Usage:     "Extract certificates from Windows Portable Executable files",
+		UsageText: `**step crypto winpe extract <file>`,
+		Description: `**step crypto winpe extract** extract certificate from a Windows Portable Executable file in PEM format.
+
+For examples, see **step help crypto winpe**.
```

Comment:
There are no examples in `step her crypto winpe`

Reply:
Good catch. Done!